packages/linux-driver-addons/dvb/depends/media_tree/package.mk

```diff
@@ -50,4 +50,19 @@ unpack() {
     cp -a "$(kernel_path)/drivers/media/v4l2-core/videobuf-res.c" "$PKG_BUILD/drivers/media/v4l2-core/"
     cp -a "$(kernel_path)/include/media/videobuf-res.h" "$PKG_BUILD/include/media/"
   fi
+
+  if listcontains "$ADDITIONAL_DRIVERS" "avl6862-aml"; then
+    mkdir -p  "$PKG_BUILD/drivers/media/dvb-avl/"
+    # Copy avl6862 driver
+    cp -a $(kernel_path)/drivers/amlogic/dvb-avl "$PKG_BUILD/drivers/media/"
+    echo "obj-y += dvb-aml/" >> "$PKG_BUILD/drivers/media/Makefile"
```

Comment:
The issue  was only in community build. With line is allow to build dvb-aml only for devices with avl6862-aml driver.